services/cinema_hall.py

```diff
@@ -0,0 +1,15 @@
+from db.models import CinemaHall
+
+
+def get_cinema_halls(cinema_halls: int) -> CinemaHall:
+    return CinemaHall.objects.all()
+
+
+def create_cinema_hall(hall_name: str,
+                       hall_rows: int,
+                       hall_seats_in_row: int) -> CinemaHall:
+    return CinemaHall.objects.create(
+        hall_name=hall_name,
+        hall_row=hall_rows,
+        hall_seats_in_row=hall_seats_in_row
```

Comment:
The parameter names used in `CinemaHall.objects.create()` do not match the field names in the `CinemaHall` model. They should be `name`, `rows`, and `seats_in_row` to match the model's fields.